example/settings.py

```diff
@@ -65,7 +65,7 @@
     'two_factor',
     'example',
 
-    'debug_toolbar',
+    # 'debug_toolbar',
```

Comment:
It doesn't. runserver ends with an error `NoReverseMatch: u'djdt' is not a registered namespace`

Seems like making it work should be part of another PR?